csv_provider/README.md

```diff
@@ -0,0 +1,31 @@
+# VSS Provider
+The aim of this script to provide and replay VSS data into a `kuksa.val` databroker. It is possible to define a sequence of VSS signals including timing delays in a CSV-file and then to apply this sequence towards a `kuksa.val` databroker with the provider. 
+
+## Arguments
+You can start the provider with the following arguments on a command line:
+
+| short argument | long argument | description | default value |
+|---- | ---- | ----- | ----|
+|-f| --file | This indicates the CSV-file containing the signals to update in the `kuksa.val` databroker. | signals.csv |
+| -a | --address | This indicates the address of `kuksa.val` databroker to connect to. | 127.0.0.1 |
+| -p | --port | This indicates the port of the `kuksa.val` databroker to connect to. | 55555 |
+| -i | --infinite | If the flag is set, the provider loops over the file until stopped, otherwise the file gets processed once. | not present/False
+| -l | --log | This sets the logging level. Possible values are: DEBUG, INFO, DEBUG, WARNING, ERROR, CRITICAL | WARNING
```

Comment:
The list of possible values is a bit difficult to understand. Is it deliberate that DEBUG and WARNING occurs twice?

Reply:
No, having DEBUG twice was simply an error. 

To align with the dbcfeeder, I also set the default logging level to DEFAULT and downgraded some of the logging.

Reply:
After reconsidering the logging levels, I decided to not downgrade to DEBUG when sending a signal because in DEBUG the kuksa_client library also adds an extensive log message. So keeping the level of INFO in this case allows to decide between the extensive log and the simple information that the line in the CSV-file has been processed.